src/gui/src/PreferencesDialog_UI.ui

```diff
@@ -564,6 +564,32 @@
            </item>
           </layout>
          </item>
+         <item>
+          <widget class="QLabel" name="panLawLbl">
+           <property name="text">
+            <string>Pan Law</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QComboBox" name="panLawComboBox">
+           <item>
+            <property name="text">
+             <string>Balance Law (0 compensation)</string>
+            </property>
+           </item>
+           <item>
+            <property name="text">
+             <string>Costant Power</string>
```

Comment:
seems to be a typo here (Costant->Co**n**stant)

Reply:
thanks